cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/Transactions.java

```diff
@@ -44,6 +50,21 @@ public static void invalidateQuietly(TransactionSystemClient txClient, Transacti
     }
   }
 
+  /**
+   * Handy method to create {@link TransactionExecutor} (See TEPHRA-71).
+   */
+  public static TransactionExecutor createTransactionExecutor(TransactionExecutorFactory factory,
+                                                              Iterable<? extends TransactionAware> txAwares) {
+    return factory.createExecutor(Iterables.transform(txAwares, Functions.<TransactionAware>identity()));
```

Comment:
What is the point of the identify transformation? Is it because createExecutor() wants a TransactionAware, not ? extends TransactionAware? If so, we should fix that in Tephra - can you open a Jira?